andela/hc-silicon
# #157694752 Configure Periodic Reports for Job Status

#### What does this PR do?
Configures periodic reports to be either daily, weekly or monthly and creates a dashboard to view all reports.
#### Description of Task to be completed?
As a user I want to be able to decide when to receive the reports, whether daily, weekly or monthly
As a user I want a view of my reports on the dashboard
#### How should this be manually tested?
After logging in, you can access the `Check Report` link under your Account Settings where you will not find any report on the dashboard. 
After setting checks you will see the reports for the set checks on the reports dashboard.

Additionally, you could also select either daily, weekly, monthly or no report as the option you want, then you will receive an email for the report indicated.
#### What are the relevant pivotal tracker stories?
[#157694752](https://www.pivotaltracker.com/story/show/157694752)
#### Screenshots 
Sample of changed report settings
![image](https://user-images.githubusercontent.com/31405837/41153581-d03c80ee-6b1f-11e8-8116-082277a34804.png)

Sample of report email sent
![image](https://user-images.githubusercontent.com/31405837/41153443-564ffa36-6b1f-11e8-8b1b-cd4e586d0dd1.png)

Sample of new reports dashboard
![image](https://user-images.githubusercontent.com/31405837/41153565-c5b7bca6-6b1f-11e8-96f2-6b486eb8a925.png)
